apps/web/src/modules/profile/edit/actor/resume/components/ResumeSlot/hooks/useResumeSlot/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { useCallback, useState } from 'react'
+
+const useResumeSlot = (
+  isFirst: boolean,
+  resumeId: number,
+  handleCancel: (resumeId: number) => void,
+) => {
+  const [isEdit, setEdit] = useState(!!isFirst)
```

Comment:
why `!!isFirst` when it is not optional?

Reply:
Copy and paste syndrome 😂